superset/views/base_api.py

```diff
@@ -122,6 +122,9 @@ def _get_related_filter(self, datamodel, column_name: str, value: str) -> Filter
             )
         return filters
 
+    def incr_stats(self, action: str, func_name: str) -> None:
+        self.stats_logger.incr(f"{action}_{self.__class__.__name__}.{func_name}")
```

Comment:
Nit: in this case, I would suggest the following structure:
```
{self.__class__.__name__}.{func_name}.{action}
```
Grouping by class name and function first makes it easier to perform glob-based selections on the consuming end.

Reply:
I'm ok with it, the goal here is to pave the way to have a consistent statsd metric name. btw on some monitoring systems you can't make glob-based selections, for example using wildcards.

Hoping to have some feedback on @etr2460 on this naming convention, the current one does not diverge a lot from the current metric names, that's why the action comes first